fibonacci.rb

```diff
@@ -5,4 +5,4 @@
   fibo = arr[-1] + arr[-2]
   arr << fibo
 end
-puts arr
\ No newline at end of file
+puts arr
```

Comment:
Compare your and this program and understand, what mistake you had done

Reply:
this is my program